crates/aptos-rest-client/src/client_builder.rs

```diff
@@ -78,6 +78,14 @@ impl ClientBuilder {
         Ok(self)
     }
 
+    pub fn api_key(mut self, api_key: &str) -> Result<Self> {
```

Comment:
Thanks @banool! Btw, is this different from setting the header 'x-api-key: \<KEY>\'?

Reply:
Yeah, that header is deprecated so I'd rather we not support it in the CLI at this point.